vignettes/basic-use.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: "Basic Use"
+author: "Mark Ewing"
+date: "2019-04-02"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Creating a Requirements File
+
+Because a requirements file is a simple text file, It's easy to create.
+
+#### From RStudio
+
+File > New File > Text File
+Then save the file as requirements.txt
+
+#### Linux and Unix-like Systems (e.g. Macs)
+
+touch requirements.txt
+
+## Adding Your First Requirement
+
+Requirements are specified one per line. The last line of the file should be empty.
+
+### CRAN Packages
+
+Packages available through the standard CRAN can be specified as an unversioned or as a versioned requirement. Unversioned requirements will always have the most recent version of the package installed. Versioned requirements will install the most recent version of a package which satisfies requirements.
+
+* Unversioned: (package)
+* Versioned: (package) (comparison) (version)
```

Comment:
I think working from an example of requirements would make things more obvious here

Reply:
Added an example in 1d42077c9608496abc2100e4255d34f6d5d5b8cf